pyiron_base/job/generic.py

```diff
@@ -952,7 +950,7 @@ def run_if_scheduler(self):
             self._logger.warning("Job aborted")
             self.status.aborted = True
             raise ValueError("run_queue.sh crashed")
-        s.logger.debug("submitted %s", self.job_name)
+        state.logger.debug("submitted %s", self.job_name)
```

Comment:
Here `state.logger` and `self._logger` are basically the same!

Reply:
I guess `self._logger` is now redundant.

Reply:
Ah yeah, up above we have `self._logger = state.logger`. I guess I was a little too much on autopilot doing the refactor. I would actually suggest to eliminate `self._logger` in a housekeeping PR later. The extra layer of misdirection is not necessary in the code base, and it's a private attribute anyhow so no one's notebooks should be accessing the logger this way anyhow.